scripts/timing.py

```diff
@@ -103,11 +103,20 @@ def time_call(cmd):
     """
 
     start = datetime.datetime.now()
-    call(cmd)
+    call_or_error(cmd)
     end = datetime.datetime.now()
 
     return (end - start).total_seconds()
 
+def call_or_error(cmd):
+    """
+        Call the given command (a list of strings representing command and args),
+        raising an error if it fails.
+    """
+    return_code = call(cmd)
+    if return_code != 0:
```

Comment:
yes thank you!

```
'Command %s exited with exit code %d'
```
?

or actually, given that this is python 3 now:
```
'Command %s exited with exit code %d'.format(cmd, return_code)
```

Reply:
done!